vwb/app/controllers/edithomepages_controller.rb

```diff
@@ -0,0 +1,56 @@
+class EdithomepagesController < ApplicationController
```

Comment:
fulfill the acceptance criteria that the admin could change the home pages

Reply:
Will add that criteria and push changes